app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -1115,7 +1176,7 @@ class StatePlayerRecyclerViewAssembler private constructor(
      */
     fun hasConversationView(hasConversationView: Boolean): Builder {
       this.hasConversationView = hasConversationView
-      featureSets += PlayerFeatureSet(showCongratulationsOnCorrectAnswer = true)
+      featureSets += PlayerFeatureSet(showCelebrationOnCorrectAnswer = true)
```

Comment:
This actually seems wrong--please remove it. I'm pretty sure this flow, as is, could lead to crashes (e.g. if we set hasConversationView without setting the Konfetti view). Alas, we should probably be storing whether conversation view is enabled using feature sets rather than a boolean, but that's outside the scope of this PR.

Reply:
Done, removed the line completely